homeassistant/components/linky/sensor.py

```diff
@@ -83,7 +104,7 @@ def __init__(self, hass, add_entities, username, password, timeout):
         )
         self.sensors.append(LinkySensor("Linky last year", self, YEARLY, INDEX_LAST))
 
-        track_time_interval(hass, self.update_linky_data, SCAN_INTERVAL)
+        async_track_time_interval(hass, self.update_linky_data, SCAN_INTERVAL)
```

Comment:
Please move all side effects out of the init method. Side effects in init method is not a good pattern. Actions, eg I/O, should be explicit and not hidden inside the init method of a class. Have a dedicated setup method that does the needed set up work of the account instance.